src/App.vue

```diff
@@ -23,6 +23,56 @@ export default {
     title: 'Welcome',
     // all titles will be injected into this template
     titleTemplate: '%s | aec.works',
+    htmlAttrs: {
+      lang: 'en',
+      amp: false,
+    },
+    meta: [
+      {
+        property: 'og:type',
+        content: 'website',
+      },
+      {
+        property: 'og:url',
+        content: 'https://aec.works',
+      },
+      {
+        property: 'og:title',
+        vmid: 'og:title',
+        template: (chunk) => `${chunk} | aec.works`,
+      },
+      {
+        property: 'og:description',
+        content:
+          'An open-source AEC community platform that promotes inclusive and enriching conversations through content-sharing and discussions.',
+      },
+      {
+        property: 'og:image',
+        content: require('@/assets/images/meta.png'),
+      },
+      {
+        property: 'twitter:card',
+        content: 'summary_large_image',
+      },
+      {
+        property: 'twitter:url',
+        content: 'https://aec.works',
+      },
+      {
+        property: 'twitter:title',
+        vmid: 'twitter:title',
+        template: (chunk) => `${chunk} | aec.works`,
+      },
+      {
+        property: 'twitter:description',
+        content:
+          'An open-source AEC community platform that promotes inclusive and enriching conversations through content-sharing and discussions.',
+      },
+      {
+        property: 'twitter:image',
+        content: require('@/assets/images/meta.png'),
```

Comment:
I haven't been too consistent with this, 
so we have a few variants of the logo. I will see if I can create a new figma page with "final" ones. I think I prefer the slightly thicker one... What do we think?

![image](https://user-images.githubusercontent.com/9513968/95005949-52b7e380-05b3-11eb-9607-c40302b4e3c0.png)
vs
![image](https://user-images.githubusercontent.com/9513968/95005955-619e9600-05b3-11eb-8e77-942088c66be5.png)

Reply:
Oh yeah, the thicker border with the the yellow looks good and consistent with other brand assets

Reply:
That 'W' alignment though 🤦
I will clean these bran assets up 
We can use any of now and will update later